src/compas_fab/backends/ros/client.py

```diff
@@ -151,7 +147,7 @@ def load_robot(self, load_geometry=False, urdf_param_name='/robot_description',
             use_local_cache = True
             path_parts = local_cache_directory.strip(os.path.sep).split(os.path.sep)
             path_parts, robot_name = path_parts[:-1], path_parts[-1]
-            local_cache_directory = os.path.sep.join(path_parts)
+            local_cache_directory = os.path.sep.join([os.sep] + path_parts)
```

Comment:
Does this seem like a reasonable fix for the problem I was having with the robot_description cache being added in the wrong directory?